### PR TITLE
Focus, Locks, and Stain updates

### PR DESCRIPTION
**Pull request by MediaResAchaea**
Well, this started as a five second attempt to update the lock data for #813 and then spiraled a bit further. From the discussion that @TheLastDarkthorne kicked off in the Achaea discord (https://discord.com/channels/872599744793419776/874411546141417492/1098763812791914518) it was pointed out that the entire concept of a Stain lock doesn't exist anymore. 

In fact, Stain hasn't blocked any cures for 7 years, and the logic within Svo was never updated to reflect this.

This PR does 4 things:

1) Updates svo.cnrl.lockdata to remove Stain locks, and updates every warning to consider Sandfever equivalent to impatience for locking purposes, as Sandfever fully blocks focus.

2) Updates the Focus illusion check and check_focus to treat sandfever as equivalent to impatience

3) Removes all traces of Stain's impact on curing Slickness, resolving 7 years of bad logic that made it so Svo wouldn't try to cure Slickness if stain was active.

4) Updates the documentation to reflect the existence of the Dragon lock warning, and make the descriptions consistent. It's not actually clear to me if this page is actually served anywhere from the in-client branch, but no harm either way I guess.

Warning: This is my first time ever touching this stuff, please validate I didn't do anything silly! They're all pretty straightforward changes, though, so hopefully it's fine!

### CODE REVIEW

File: svo (setup, misc, empty, funnies, dor).xml
Comment (MediaResAchaea) [inline svo (setup, misc, empty, funnies, dor).xml:1]: Oh, this actually isn't quite right.

Comment (MediaResAchaea) [reply svo (setup, misc, empty, funnies, dor).xml:1]: Should be right now, but would appreciate vetting!

File: svo (setup, misc, empty, funnies, dor).xml [hunk 1]
```diff
@@ -2840,9 +2840,7 @@ end
 
 empty.eat_bloodroot = function()
   svo.lostbal_herb()
-  svo.rmaff('paralysis')
-
-  if not affs.stain then svo.rmaff('slickness') end
+  svo.rmaff('paralysis', 'slickness')
```
Comment (TheLastDarkthorne) [inline svo (setup, misc, empty, funnies, dor).xml]: rmaff function only accepts one arg
just call it twice once for para and once for slick

**Comment by TheLastDarkthorne** [comment]
Sounds all good, just fix that one little thing and make sure it is all working (like focusing properly removing the affs if you got them and the lock warnings working with the sandfever changes, you can use `svo.addaff` to make svo think you have an aff)

File: svo (setup, misc, empty, funnies, dor).xml (hunk 1 above)
Comment (Sebguer) [reply svo (setup, misc, empty, funnies, dor).xml:2843]: oh, rip, i thought i copied this from somewhere but i guess it was a different function 🤦‍♂️

Comment (MediaResAchaea) [reply svo (setup, misc, empty, funnies, dor).xml]: I updated this to an array of the affs, which I think is right based on existing (which is what I was trying to copy but missed that it should be an array):

https://github.com/MediaResAchaea/svof/blob/ae640b2d4ea8214b84dc26d44558c2023486fe9d/svo%20(setup%2C%20misc%2C%20empty%2C%20funnies%2C%20dor).xml#L2921

**Comment by MediaResAchaea** [comment]
Testing log: https://ada-young.com/pastebin/HLtiJKoC

Everything looks to work as expected.

(Note: It focuses through Sandfever in the opening bit because I had forgotten to pull over the focus change when I was testing the lock warnings 😅)

**Comment by MediaResAchaea** [comment]
Not sure how I missed these, but there were also default scenarios for when to use shrugging, restore, and dragonheal to escape a stainlock.

**Comment by TheLastDarkthorne** [comment]
Really good catches here, maybe you will understand the system better than I do lol I totally overlooked that you can pass a table to the affliction functions and didn't see the scenarios thing either.

Tbh I would never thought of the stain stuff until you pointed it out lol

**Pull request merged**